charts/dispatch/charts/identity-manager/values.yaml

```diff
@@ -13,7 +13,20 @@ service:
   name: identity-manager
   type: ClusterIP
   externalPort: 80
-  internalPort: 8080
+  internalPort: 80
```

Comment:
saw you removed the reference to service.internalPort in some place and replace it with 8080. it is confusing you also use 80 here, is this your intention?

Reply:
Previously, IDM  listened on port 8080 and hence internalPort for the corresponding k8s service was also 8080.

I have now added Oauth2Proxy as a side-car which will proxy requests to Identity Manager. Hence, I need to switch internalPort to the port of Oauth2proxy (i.e standard http port). Thus, IDM's port can be directly set to 8080 since it is upstream for Oauth2proxy and not exposed to the service. Is that clear?

Reply:
sounds great. suggest that it will be even better/clear if there's some notes/naming to clarify on 
80 --> oauth2-proxy
8080 --> IDM
as mostly people think internalPort is for the main service (i.e. IDM here)

Reply:
Sure @seanhuxy Rev'ing